tasks/security_agent.py

```diff
@@ -69,3 +85,17 @@ def build(ctx, race=False, go_version=None, incremental_build=False,
     }
 
     ctx.run(cmd.format(**args), env=env)
+
+@task()
+def gen_mocks(ctx):
+    """
+    Generate mocks.
+    """
+
+    gopath = get_gopath(ctx)
+    if not os.path.exists(os.path.join(gopath, "bin/mockery")):
+        with ctx.cd(gopath):
+            ctx.run("go get -u github.com/vektra/mockery/cmd/mockery", env={'GO111MODULE': 'on'})
```

Comment:
We should do this in the `inv deps` command.

Reply:
It's a one-off tool and it shouldn't be listed in the dependencies anywhere since not used in the code base - otherwise `go mod tidy` will keep cleaning it up every time.